src/ildasm/unixcoreclrloader/coreclrloader.cpp

```diff
@@ -11,10 +12,12 @@ using namespace std;
 void *CoreCLRLoader::LoadFunction(const char *funcName)
 {
     void *func = nullptr;
-    if (coreclrLib == nullptr) {
+    if (coreclrLib == nullptr)
+    {
         fprintf(stderr, "Error: coreclr should be loaded before loading a function: %s\n", funcName);
     }
-    else {
+    else
+    {
         func = dlsym(coreclrLib, funcName);
         if (func == nullptr) {
```

Comment:
Would you mind fixing this curly brace location as well? I know it is not your change, but since you are touching the code around ...